CONTRIBUTING.md

```diff
@@ -1 +1,61 @@
+# Introduction
 
+### Welcome to Botpress!
+
+First off, thank you for considering contributing to Botpress
+
+Following these guidelines helps to communicate that you respect the time of the developers managing and developing this open source project. In return, they should reciprocate that respect in addressing your issue, assessing changes, and helping you finalize your pull requests.
+
+Botpress is an open source project and we welcome any contributions from our community! There are more ways to contribute that just fixing bugs and writing features - improving the documentation, submitting bug reports and writing tutorials are all great contributions and will be welcomed as much as any other.
+
+However as a reminder we do have a [Slack channel](https://slack.botpress.io/) and a [website](https://botpress.io/) so please use these as places to find support rather than using the issue tracker which should be reserved for feature requests and bug reports.
```

Comment:
Great point. I'm still thinking about our bot idea. It could really help us redirect such questions to the slack.